src/main/java/uk/gov/pay/api/model/directdebit/mandates/CreateMandateResponse.java

```diff
@@ -18,13 +18,7 @@
     private static final String DESCRIPTION_NAME = "description";
     private static final String PAYMENT_PROVIDER_NAME= "payment_provider";
 
-    private final String mandateId;
-    private final String providerId;
-    private final String reference;
-    private final String returnUrl;
     private final String createdDate;
-    private final MandateStatus state;
-    private final MandateLinks links;
     private final String description;
     private final String paymentProvider;
```

Comment:
Shouldn't the get mandate response have a description, payment provider and created date too?

Same comment as for the dd-connector change, is there an advantage to having 2 models over just having 1 where some fields won't be populated some of the time?

Reply:
fixed